RZBluetoothTests/RZBSimulatedTestCase.m

```diff
@@ -7,11 +7,11 @@
 //
 
 #import "RZBTestDefines.h"
-#import "RZMockBluetooth.h"
+#import "RZBluetooth/RZMockBluetooth.h"
 #import "RZBSimulatedTestCase.h"
 #import "NSRunLoop+RZBWaitFor.h"
-#import "RZBCentralManager+Private.h"
-#import "RZBLog.h"
+#import "RZBluetooth/RZBCentralManager+Private.h"
```

Comment:
It's been a while... Why do we need the private header? Ideally we wouldn't, but if we can safely remove it we should.

Reply:
Looks like it exposes the `RZBCommandDispatch *dispatch` property which is used in `waitForQueueFlush` and required for `RZBAssertCommandCount()`.